orcid-persistence/src/main/java/org/orcid/persistence/dao/impl/ProfileDaoImpl.java

```diff
@@ -746,12 +746,29 @@ public boolean deactivate(String orcid) {
     @SuppressWarnings("unchecked")
     @Override
     public String getLockedReason(String orcid) {
-        Query query = entityManager.createNativeQuery("SELECT reason_locked, reason_locked_description from profile where orcid = :orcid");
+        Query query = entityManager.createNativeQuery("SELECT reason_locked, reason_locked_description, record_locked_admin_id, record_locked_date, last_indexed_date from profile where orcid = :orcid");
         query.setParameter("orcid", orcid); 
+        // Format results
         List<Object[]> result = query.getResultList();
-        String reason = (String) result.get(0)[0];
-        String description = (String) result.get(0)[1];
-        return reason != null ? reason + (description != null ? " (" + description + ")" : "") : null;
+        String reason = " for: " + (String) result.get(0)[0];
```

Comment:
@auumgn lets do this in the manager or controller side of things, not in the DAO